airflow/sensors/sql.py

```diff
@@ -70,27 +71,13 @@ def __init__(
 
     def _get_hook(self):
         conn = BaseHook.get_connection(self.conn_id)
-
-        allowed_conn_type = {
-            'google_cloud_platform',
```

Comment:
Note: apparently there was a mistake there - > we changed the type of connection for BigQueryHook to be 'gcpbigquery` (historically it used to be 'google_cloud_platform' so this would not work for BigQuery :( - but after that change it will work fine as BigQueryHook derives from DBApi

Reply:
No no, that was just a note. It's good as you proposed, your change actually fixes the error that we had before.